dbtmetabase/metabase.py

```diff
@@ -129,6 +129,7 @@ def __init__(
         exclude_sources: bool = False,
         http_extra_headers: Optional[dict] = None,
         http_timeout: int = 15,
+        http_adapter: Optional[BaseAdapter] = None,
```

Comment:
You don't need BaseAdapter, just use HTTPAdapter. That's what your PKCS library extends as well.

Reply:
Thanks